NOTICE

```diff
@@ -0,0 +1,12 @@
+Bootloose (formerly Footloose)
```

Comment:
We should decide how the project name is spelled. We mostly use lowercase when referring to commands and files and such, but sometimes, like here, also the capitalized form. If the project name should be "Bootloose", then I'd use it capitalized in other places as well (e.g. the license headers). If it should be "bootloose", then I'd change it to lowercase here and a few other places. The latter seems to be easier.